go/wire.go

```diff
@@ -127,6 +128,12 @@ func GetBytes32(buf []byte) [32]byte {
 	return *(*[32]byte)(unsafe.Pointer(&buf[0]))
 }
 
+func GetUint256(buf []byte) *big.Int {
+	x := big.NewInt(0)
+	x.SetBytes(buf[0:FieldSizes[TypeUint256]])
```

Comment:
note that this is a copying operation; is there anyway to do this without copying?

Reply:
could not find a better way :( it is an internal go implementation.